src/middleware/routers/index.js

```diff
@@ -37,5 +38,6 @@ router.use('/v1/search', search);
 router.use('/v1/referees', referees);
 router.use('/v1/staffs', staffs);
 router.use('/v1/request-types', requestTypes);
+router.use('/v1/comments', comments);
```

Comment:
do we use this route? if not, we can remove first. it'd just introduce confusion.